### PR TITLE
Updated Ticks.md

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Updated the documentation for improved clarification of how Carbon handles ticks.

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs/docs/helpers/Ticks.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ The tick values are set one of 3 methods, in the following priority:
 
 -   **Default**
 
-    If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
+    If neither `ticksCount` or custom tick values are prvided, then the default behavior is as follows:
```
Comment (yuderekyu) [inline packages/terra-graphs/docs/helpers/Ticks.md:130]:
```suggestion
    If neither `ticksCount` or custom tick values are provided, then the default behavior is as follows:
```

File: packages/terra-graphs/docs/helpers/Ticks.md [hunk 2]
```diff
@@ -127,6 +127,8 @@ The tick values are set one of 3 methods, in the following priority:
...
+    
+    - If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
 
-    If there is no Y2 axis, the Carbon will use D3.js to automatically find the tick values.
+    - If there is no Y2 axis and `ticksCount = undefined`, the Carbon will use D3.js to automatically find the tick values.
```
Comment (yuderekyu) [inline packages/terra-graphs/docs/helpers/Ticks.md:134]:
```suggestion
    - Else if there is no Y2 axis and `ticksCount = undefined`, Carbon will use D3.js to automatically find the tick values.
```

File: packages/terra-graphs/docs/helpers/Ticks.md [hunk 3]
```diff
@@ -127,6 +127,8 @@ The tick values are set one of 3 methods, in the following priority:
 
 -   **Default**
 
-    If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
+    If neither `ticksCount` or custom tick values are prvided, then the default behavior is as follows:
+    
+    - If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
```
Comment (yuderekyu) [inline packages/terra-graphs/docs/helpers/Ticks.md:132]:
```suggestion
    - If the Y2 Axis is visible, then Carbon will calculate and use a ticksCount value, based on the ranges of Y and Y2.
```

File: packages/terra-graphs/docs/helpers/Ticks.md (hunk 2 above)
Comment (yuderekyu) [inline packages/terra-graphs/docs/helpers/Ticks.md:134]: is it possible to not fall under either of these conditions?

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Review by mraplumb — APPROVED** [review]
+1 to Derek's suggestions

File: packages/terra-graphs/docs/helpers/Ticks.md (hunk 2 above)
Comment (mraplumb) [reply packages/terra-graphs/docs/helpers/Ticks.md:134]: I don't think there is a way for a consumer to get past all of these branches of logic.  Any graph with 1 or 2 Y-axes will qualify for at least one of these logic checks

**Review by yuderekyu — APPROVED** [review]

**Pull request merged**